### PR TITLE
gitAndTools.git-remote-codecommit: init at 1.15.1

### PR DESCRIPTION
**Pull request by zaninime**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/git-remote-codecommit/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  pname = "git-remote-codecommit";
+  version = "1.15.1";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "052j1a4a9gccyq6wrzih81jgxx9ayqqdh42iy72zibgrji5d96fd";
+  };
+
+  propagatedBuildInputs = [ botocore ];
```
Comment (jonringer) [inline pkgs/development/python-modules/git-remote-codecommit/default.nix:13]: There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Comment (zaninime) [reply pkgs/development/python-modules/git-remote-codecommit/default.nix]: The source contains 23 tests. Switching to the GitHub repo fixed it.

**Comment by zaninime** [comment]
@jonringer feel free to have a look again whenever you can.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 98847 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>gitAndTools.git-remote-codecommit</li>  </ul></details>

**Pull request merged**